### ISSUE TITLE
(entire) Canvas wont render if Blood-n-Guts is installed but a GM is not logged in

### ISSUE THREAD

**Issue by 736dave** [edzillion/blood-n-guts#230]
If I 'join as' a player, the canvas won't render.  If I remove Blood-n-Guts then refresh, the canvas renders normally.

**Comment by edzillion** [comment]
That's normal. You need a GM to be present for Blood N Guts to work!

**Comment by 736dave** [comment]
> That's normal. You need a GM to be present for Blood N Guts to work!

Yes but it seems to crash the rest of the canvas as well. I figure players should still be able to see the welcome scene, without the blood layer ... at the moment it just goes grey.

**Comment by yoshikidneo** [comment]
I agree with 736dave, a toggle for certain scenes would be really neat so that players can log into the server without a GM and see the landing page/world maps.

**Comment by 736dave** [comment]
> I agree with 736dave, a toggle for certain scenes would be really neat so that players can log into the server without a GM and see the landing page/world maps.

... or just have it fail gracefully and continue on with drawing the rest of the canvas _without_ the B&G layer ?

**Comment by Doresain** [comment]
> > I agree with 736dave, a toggle for certain scenes would be really neat so that players can log into the server without a GM and see the landing page/world maps.
> 
> ... or just have it fail gracefully and continue on with drawing the rest of the canvas _without_ the B&G layer ?

Yes, this. This bug not only makes it difficult to let players join the server and wait in a lobby, it also makes it harder to test my own table by logging in as a player.

**Comment by LastElf** [comment]
Adding my "Me too" to this

**Comment by Nordiii** [comment]
Yeah just wanted to test something as player without GM online and the scene would not render... Totaly fine if there is no blood while the GM is offline, but not rendering the scene is big.

**Comment by FGiorgi** [comment]
I agree with everybody else. I have to deactivate blood'n guts every time I log off as a GM otherwise my players cannot see anything if I'm not online. I have a game with a lot of splash screens that the players have access to with links to rules, lore, characters, etc... and them being able to join and search for stuff when I'm offline is paramount.
